### PR TITLE
feat(ras-acc): add faq section to otp email

### PR DESCRIPTION
**Pull request by chickenn00dle**
### All Submissions:

* [ ] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/trunk/.github/CONTRIBUTING.md)?
* [ ] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

**PR 3 of 4**

This PR adds onto the work in https://github.com/Automattic/newspack-plugin/pull/3032 and https://github.com/Automattic/newspack-plugin/pull/3033 by adding an FAQ section to the otp email:

Screenshot 2024-04-05 at 13 16 04

### How to test the changes in this Pull Request:

1. With these changes checked out, go to Newspack > Engagment > Show Advanced Settings and select the one-time password email under Transactional Email Content section
2. Trash the post (so that we can force our updated template to be used on next otp sign in attempt)
3. As a logged out user, select the sign in link and attempt to login via email to trigger an otp email
4. Check the otp email and confirm the FAQ section is present and the link to set a new password works as expected

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by adekbadek — CHANGES_REQUESTED** [review]
The update logic discussion (https://github.com/Automattic/newspack-plugin/pull/3032#pullrequestreview-1982974830) applies here, too. 

Also, I'm confused about structure of these PRs (PRs to other PR branches, instead of PRs to `trunk`). Should there be an `epic` branch?

**Comment by chickenn00dle** [comment]
Apologies for the confusion @adekbadek!

I meant to update the descriptions for these email PRs, but that slipped my mind. I've broken the changes up into 4 different PRs, and since they all sort of rely on one another they are based off of the other (all going into epic/ras-acc branch). So there is an order which I've updated each of the PR descriptions with.

But to be clear, here is the order for these PRs:

PR 1 - Already approved
PR 2- https://github.com/Automattic/newspack-plugin/pull/3033
PR 3 - This one
PR 4 - https://github.com/Automattic/newspack-plugin/pull/3038

File: includes/templates/reader-activation-emails/otp.php [hunk 1]
```diff
@@ -18,39 +18,51 @@
...
+	<!-- wp:heading {"level":3,"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}}} -->
+	<h3 class="wp-block-heading" style="font-style:normal;font-weight:400">' . __( 'Frequently Asked Questions', 'newspack' ) . '</h3>
+	<!-- /wp:heading -->
+	<!-- wp:paragraph -->
+	<p><strong>' . __( 'What is a magic link?', 'newspack' ) . '</strong><br>' . __( 'It\’s a temporary, password-free link like the one above to help you quickly and securely verify your identity and sign in – all without needing to set a password.', 'newspack' ) . '</p>
+	<!-- /wp:paragraph -->
+	<!-- wp:paragraph -->
+	<p><strong>' . __( 'Can I still create a password?', 'newspack' ) . '</strong><br>' . sprintf( /* Translators: 1: opening HTML anchor tag 2: closing HTML anchor tag */ __( 'Yes. For security reasons, we recommend signing in with Google, Facebook, or Apple, or using a one-time code sent to your email, but you can set a password if you prefer. %1$sClick here%2$s to create a password.', 'newspack' ), '<a href="*SET_PASSWORD_LINK*" data-type="link" data-id="*SET_PASSWORD_LINK*">', '</a>' ) . '</p>
```
Comment (adekbadek) [inline includes/templates/reader-activation-emails/otp.php]: We don't support signing in with Facebook and Apple yet. This should only mention Google.

**Review by adekbadek — APPROVED** [review]
Pre-approving pending a small email content change.

File: includes/templates/reader-activation-emails/otp.php (hunk 1 above)
Comment (chickenn00dle) [reply includes/templates/reader-activation-emails/otp.php]: Good catch! Removed in https://github.com/Automattic/newspack-plugin/pull/3034/commits/51015903b08ab45e69982a34997e127c696fa3bd

**Pull request merged**